### PR TITLE
Address master rename

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by greg0ire** [comment]
I have some doubts about this: 

https://github.com/doctrine/DoctrineMongoDBBundle/blob/30f3d4aa697ac5d8322f27b0a154e66e43212e81/composer.json#L57-L61

https://github.com/doctrine/DoctrineMongoDBBundle/blob/7992f675e83fd52d0f0a42899c1946f697518ed9/.doctrine-project.json#L6-L15

Is `master` really supposed to be 5? Or just 4.4? Since you have a `4.3.x` branch but no 4.3.0 that would be a bit weird, but it's better to check I suppose.

**Comment by SenseException** [comment]
I can't tell by the diff to master if the intended version was a major one.

**Comment by greg0ire** [comment]
Maybe we should merge master into 4.3.x haha

**Comment by malarzm** [comment]
As far as I can tell there are no plans for 5.x. It's safe to assume that master will become 4.4 at some point.

**Comment by greg0ire** [comment]
I forgot about this. Since [we voted to freeze the migration away from master](https://app.mieuxvoter.fr/result/O9aUy3s3s899fp0AEtlB), let's close this for now. I removed `5.0.x`

> It's safe to assume that master will become 4.4 at some point.

Then the branch-alias on that branch should be fixed.

**Comment by alcaeus** [comment]
> As far as I can tell there are no plans for 5.x. It's safe to assume that master will become 4.4 at some point.

That is correct, no 5.x is currently planned. The next minor release will be in the 4.x major version.

**Comment by greg0ire** [comment]
The migration away from `master` is resumed.

Todo for @alcaeus or @malarzm :
- [x] use Github's rename feature to rename `master` to 4.4.x
- [x] merge this
- [x] merge 4.3.x up into 4.4.x

File: .doctrine-project.json [hunk 1]
```diff
@@ -3,6 +3,12 @@
     "shortName": "DoctrineMongoDBBundle",
     "slug": "doctrine-mongodb-bundle",
     "versions": [
+        {
+            "name": "4.4",
+            "branchName": "4.4.x",
```
Comment (malarzm) [inline .doctrine-project.json:8]: Shouldn't this happen after we release 4.3.0 and have 4.4.x branch in place?

Comment (greg0ire) [reply .doctrine-project.json:8]: `master` can be renamed to 4.4.x right now, there is no need for releasing 4.3.0 for that, is there? It's the first item of the todo list :slightly_smiling_face:

Comment (malarzm) [reply .doctrine-project.json:8]: Dang, this is about renaming master to 4.4.x first 🤦‍♂️

Comment (malarzm) [reply .doctrine-project.json:8]: @greg0ire yeah sorry, I went straight to the review from first email and haven't seen the todo list :)

Comment (greg0ire) [reply .doctrine-project.json:8]: @malarzm let's proceed? I can do it if you don't feel comfortable with that, but I'd rather you approve this PR before going through the todo list myself.

Comment (malarzm) [reply .doctrine-project.json:8]: Sorry for the lag, sadly I didn't find as much time for ODM last week as I wanted. Going through the checklist now :)

**Review by malarzm — APPROVED** [review]

**Comment by malarzm** [comment]
Thanks @greg0ire for preparing this!

**Comment by malarzm** [comment]
> merge 4.3.x up into 4.4.x

I'll let our automation handle that after releasing 4.3.0 :)

**Pull request merged**